ui/ducks/send/send.test.js

```diff
@@ -1443,6 +1469,13 @@ describe('Send Slice', () => {
               address: 'Address',
```

Comment:
^^^ This test should probably be renamed: 'should create actions to reset recipient input and ens, calculate gas and then validate input'

Reply:
Done in 2c6dd9b8d